app/src/main/java/prestin/l/lightcontrol/View/Activity/SettingsActivity.java

```diff
@@ -118,7 +121,7 @@ public void onBuildHeaders(List<Header> target) {
     }
 
     protected boolean isValidFragment(String fragmentName) {
-        return PreferenceFragment.class.getName().equals(fragmentName)
+        return  PreferenceFragment.class.getName().equals(fragmentName)
```

Comment:
Leerzeichen sollte weg

```suggestion
        return PreferenceFragment.class.getName().equals(fragmentName)
```